cases/arithmetic_decimal/bitwise_and.yaml

```diff
@@ -0,0 +1,121 @@
+base_uri: https://github.com/substrait-io/substrait/blob/main/extensions/substrait/extensions/functions_arithmetic_decimal.yaml
+function: bitwise_and
+cases:
+  - group:
+      id: basic
+      description: Basic examples without any special cases
+    args:
+      - value: 0
+        type: decimal<1, 0>
+      - value: 1
+        type: decimal<1, 0>
+    result:
+      value: 0
+      type: decimal<1, 0>
+  - group: basic
+    args:
+      - value: 127
+        type: decimal<3, 0>
+      - value: 127
+        type: decimal<3, 0>
+    result:
+      value: 127
+      type: decimal<3, 0>
+  - group: basic
+    args:
+      - value: -127
+        type: decimal<3, 0>
+      - value: -10
+        type: decimal<2, 0>
+    result:
+      value: -128
+      type: decimal<3, 0>
+  - group: basic
+    args:
+      - value: 31766
+        type: decimal<5, 0>
+      - value: 900
+        type: decimal<3, 0>
+    result:
+      value: 4
+      type: decimal<5, 0>
+  - group: basic
+    args:
+      - value: -31766
+        type: decimal<5, 0>
+      - value: 900
+        type: decimal<3, 0>
+    result:
+      value: 896
+      type: decimal<5, 0>
+  - group: basic
+    args:
+      - value: 2147483647
+        type: decimal<10, 0>
+      - value: 1234567
+        type: decimal<7, 0>
+    result:
+      value: 1234567
+      type: decimal<10, 0>
+  - group: basic
+    args:
+      - value: 2147483647
```

Comment:
should this be -2147483647?

Otherwise it's a duplicate of the case above.

Reply:
fixed it, Thanks